plugins/kubernetes-react/package.json

```diff
@@ -6,9 +6,22 @@
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "publishConfig": {
-    "access": "public",
-    "main": "dist/index.esm.js",
-    "types": "dist/index.d.ts"
+    "access": "public"
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./api": "./src/api/index.ts",
```

Comment:
This is the most important part.
When this path export is used in `kubernetes` package, it allows only the APIs be loaded synchronously, and all the react stuff is dynamically loaded when kubernetes plugin is navigated to.